text/0201-scope-attributes.md

```diff
@@ -38,28 +38,80 @@ The following is the summary of proposed changes:
 
 ### API Changes
 
+#### Tracer
+
 `Get a Tracer` API will be extended to add the following parameter:
 
 ```
 - `attributes` (optional): Specifies the instrumentation scope attributes to associate
   with emitted telemetry.
 ```
 
-`Get a Metter` API will be extended to add the following parameter:
+Since the attributes are optional this is a backwards compatible change.
+
+We will modify the following clause:
+
+```
+It is unspecified whether or under which conditions the same or different
+`Tracer` instances are returned from this functions.
+```
+
+and replace it by:
+
+```
+The implementation MUST NOT return the same `Tracer` when called repeatedly with
+different values of parameters.
```

Comment:
Why is this important? Are you trying to encourage users to create a "Tracer" for every request to add runtime properties there?

Please consider to write the considerations/use-cases for making this decision.

Reply:
I [expanded](https://github.com/open-telemetry/oteps/pull/201/commits/5e50e420286c69f7c093139820daae6e945c28b8) the code example in the OTEP to also demonstrate this.